### PR TITLE
chore: [CO-334] cleanup Zlicense

### PR DESCRIPTION
**Pull request by keshavbhatt**
**Key changes:** 

* remove ZLicense related classes from JAXB registry
   - clean related tests from Jaxb Unit Tests

* cleanup: remove ZLicense related elements from GetInfo SOAP API

   - clean GetInfoResponse Tests for License Elements

* cleanup: remove ZLicense related handlers from AdminService

* cleanup: remove ZLicense related Constants

   - clean AccountConstants

   - clean AdminConstants

   - clean GQLConstants

* cleanup: remove ZLicense-related dead code

   - remove related services &SOAP APIs that no longer required.

   - Remove services and endpoints: ActivateLicense, CheckLicense, LicenceInfo,InstallLicense, LicenceStatus etc

* cleanup: delete ZLicense class

* chore: remove unused methods

   - remove unused method getLicenses

**How to check the changes:**
 Use your IDE diff tool (ignoring imports and formatting) to check the diffs instead Github's webview

### CODE REVIEW

**Review by frisonisland — APPROVED** [review]

File: store/src/java/com/zimbra/qa/unittest/TestProvDomainStatus.java [hunk 1]
```diff
@@ -40,1054 +30,1468 @@
...
-                eId.setText(mCtx.mDomainId);
-                Element eStatus = req.addElement(AdminConstants.E_STATUS);
-                eStatus.setText(curStatus);
-                mCtx.mSoapClient.invoke(req);
-            }
-        }
+        String key = R.getKey(code, msg);
+        Result r = R.sMap.get(key);
+        assertNotNull(key, r);  // unknown exception
         */
```
Comment (Polpetta) [inline store/src/java/com/zimbra/qa/unittest/TestProvDomainStatus.java:727]: Please if you can remove those commented lines

File: store/src/java/com/zimbra/qa/unittest/TestProvDomainStatus.java [hunk 2]
```diff
@@ -40,1054 +30,1468 @@
...
-                String msg = e.getMessage();
-
-                assertNotNull(code);
-                assertNotNull(msg);
-
-                /*
-                String key = R.getKey(code, msg);
-                Result r = R.sMap.get(key);
-                assertNotNull(key, r);  // unknown exception
-                */
```
Comment (Polpetta) [inline store/src/java/com/zimbra/qa/unittest/TestProvDomainStatus.java:717]: Same

File: store/src/java/com/zimbra/qa/unittest/TestProvDomainStatus.java [hunk 3]
```diff
@@ -40,1054 +30,1468 @@
...
-
-        dispatcher.registerHandler(AdminConstants.ADD_ACCOUNT_LOGGER_REQUEST, new AddAccountLogger());
-        dispatcher.registerHandler(AdminConstants.REMOVE_ACCOUNT_LOGGER_REQUEST, new RemoveAccountLogger());
-        dispatcher.registerHandler(AdminConstants.GET_ACCOUNT_LOGGERS_REQUEST, new GetAccountLoggers());
-        dispatcher.registerHandler(AdminConstants.GET_ALL_ACCOUNT_LOGGERS_REQUEST, new GetAllAccountLoggers());
-
-        dispatcher.registerHandler(AdminConstants.CHECK_DIRECTORY_REQUEST, new CheckDirectory());
-
-        dispatcher.registerHandler(AdminConstants.FLUSH_CACHE_REQUEST, new FlushCache());
-         */
```
Comment (Polpetta) [inline store/src/java/com/zimbra/qa/unittest/TestProvDomainStatus.java:937]: I don't think we need those too

File: store/src/java/com/zimbra/qa/unittest/TestProvDomainStatus.java [hunk 4]
```diff
@@ -40,1054 +30,1468 @@
...
+              + (cause == null
+                  ? ""
+                  : " (cause: " + cause.getClass().getName() + " " + cause.getMessage() + ")"));
+      e.printStackTrace(System.out);
+      System.out.println("\nTest " + TEST_ID + " failed!");
+    } /* catch (AssertionFailedError e) {
+          System.out.println("\n===== assertion failed =====");
+          System.out.println(e.getMessage());
+          e.printStackTrace(System.out);
+      } */
```
Comment (Polpetta) [inline store/src/java/com/zimbra/qa/unittest/TestProvDomainStatus.java:1491]: Also here

**Review by Polpetta — APPROVED** [review]
LGTM there's more code we can remove if you will. Checked with IntelliJ

**Comment by keshavbhatt** [comment]
@Polpetta done!

**Pull request merged**